### PR TITLE
Fixed Test Warning: Please use `require("history").createHashHistory

### PR DESCRIPTION
**Pull request by debbieellis7**
## Description

Issue: #905

Fixed test deprecation code warning in the following file:
- tests/mocha/unit-tests/components/CoreFonts/core.font.container.spec.js
- tests/mocha/unit-tests/components/CoreFonts/core.font.list.results.spec.js
- tests/mocha/unit-tests/components/Template/template.activate.button.spec.js
- tests/mocha/unit-tests/components/Template/template.button.spec.js
- tests/mocha/unit-tests/components/Template/template.close.dialog.spec.js

## Testing instructions

Run: yarn test

## Checklist:
- [x] I've tested the code.
- [x] My code is easy to read, follow, and understand
- [x] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation / docblocks.

### CODE REVIEW

File: tests/mocha/unit-tests/components/CoreFonts/core.font.container.spec.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ describe('<CoreFontContainer />', () => {
     consoleList = {}
     retry = []
 
-    mock.post('/saveCoreFont', (req) => {
```
Comment (jakejackson1) [inline tests/mocha/unit-tests/components/CoreFonts/core.font.container.spec.js]: The removal of `req` appears unrelated to this PR. If that's the case, the change should be made in a different PR.

Comment (debbieellis7) [reply tests/mocha/unit-tests/components/CoreFonts/core.font.container.spec.js]: Its required after running the phpstorm format code. And as I checked it is also no used elsewhere.

Comment (jakejackson1) [reply tests/mocha/unit-tests/components/CoreFonts/core.font.container.spec.js]: Yes, but it's not relevant to this PR. As a rule, each PR / commit should have one purpose. In this case it's fixing up the hash history error message and anything unrelated to that should not be included.

Comment (debbieellis7) [reply tests/mocha/unit-tests/components/CoreFonts/core.font.container.spec.js]: ok
